wondrous-app/components/organization/members/styles.tsx

```diff
@@ -1,20 +1,229 @@
+import Grid from '@mui/material/Grid';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import Select from '@mui/material/Select';
+import MenuItem from '@mui/material/MenuItem';
+import SearchIcon from 'components/Icons/search';
 import palette from 'theme/palette';
 
+export const MembersWrapper = styled(Grid)`
+  && {
+    padding: 25px 0;
+    width: 95%;
+    margin: 0 auto;
+  }
+`;
+
+export const MembersHeader = styled(Grid)`
+  && {
+    display: flex;
+    align-items: baseline;
+    flex-direction: column;
+    gap: 18px;
+    padding-bottom: 24px;
+    border-bottom: 1px solid ${palette.grey900};
+  }
+`;
+
+export const MembersHeading = styled(Typography)`
+  && {
+    font-family: 'Space Grotesk';
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 18px;
+    letter-spacing: 0.03em;
+    color: ${palette.white};
+  }
+`;
+
+export const UserRole = styled(Grid)`
+  && {
+    display: flex;
+    align-items: center;
+    gap: 8px;
+    padding: 3px 5px 3px 3px;
+    background: ${palette.grey900};
+    border: 1px solid ${(props) => props.borderColor};
+    border-radius: 1000px;
+
+    div {
+      padding: 2px 7px;
+    }
+
+    p {
+      font-size: 13px;
+      font-weight: 500;
+    }
+  }
+`;
+
+export const FilteredRolesContainer = styled(Grid)`
+  && {
+    display: flex;
+    align-items: center;
+    gap: 16px;
+
+    p {
+      font-size: 13px;
+      font-weight: 500;
+      text-transform: capitalize;
+    }
+  }
+`;
+
+export const FilterMembersContainer = styled(Grid)`
+  && {
+    position: relative;
+    display: flex;
+    align-items: center;
+    gap: 14px;
+    padding: 24px 0;
+    border-bottom: 1px solid ${palette.grey900};
+  }
+`;
+
+export const FilterMembersInput = styled.input`
+  flex: 1;
+  background: ${palette.black101};
+  padding: 12px 14px 12px 42px;
+  border: 1px solid ${palette.black101};
+  border-radius: 1000px;
+  outline: none;
+  font-family: 'Space Grotesk';
+  font-size: 15px;
+  font-weight: 400;
+  color: ${palette.grey250};
+  transition: border-color 0.3s ease-in-out;
+
+  &:focus {
+    border: 1px solid ${palette.black81};
+  }
+`;
+
+export const FilterMembersInputIcon = styled(SearchIcon)`
+  && {
+    position: absolute;
+    top: 50%;
+    left: 1em;
+    transform: translateY(-50%);
+    pointer-events: none;
+
+    path {
+      stroke: ${palette.white};
+    }
+  }
+`;
+
+const StyledSelect = styled(Select)`
+  && {
+    background: ${palette.black101};
+    color: ${palette.white};
+    height: 45px;
+    width: 245px;
+    border-radius: 6px;
+    border: 1px solid transparent;
+    transition: border 0.2s ease-out;
+
+    :hover {
+      border: 1px solid ${palette.grey79};
+    }
+
+    svg {
+      color: ${palette.white};
+      transition: transform 0.2s ease-out;
+    }
+
+    p {
+      margin: 0;
+    }
+  }
+`;
+
+export const RoleFilterSelect = styled(({ className, ...props }) => (
```

Comment:
I think we shouldn't be building selects from scratch at this point? @Lamperoyge ?

Reply:
true, we need to use already existing ones. overall we need to unify our UI components @SankhaChak ping me on discord I can help if needed